rust-runtime/aws-smithy-runtime-api/src/client/identity.rs

```diff
@@ -159,6 +159,14 @@ pub trait ResolveIdentity: Send + Sync + Debug {
     fn fallback_on_interrupt(&self) -> Option<Identity> {
         None
     }
+
+    /// Returns the location of an identity cache associated with this identity resolver.
+    ///
+    /// By default, identity resolvers will use the identity cache stored in runtime components.
+    /// Implementing types can change the cache location if they want to.
```

Comment:
Would be good to state why an implementation might want to change the location.

Reply:
Added an extra sentence in the comment (457f6b8).